### PR TITLE
Initial version of the landing page

### PR DESCRIPTION
**Pull request by wearethefoos**
Hi @gandjhar check out the new homepage I just made.

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>CatPet</h1>
+<p>Welcome to CatPet!</p>
+<p><small>Coming Soon!</small></p>
+<p>
+  <%= image_tag "cat.jpg", class: "cat" %>
```
Comment (wearethefoos) [inline app/views/welcome/index.html.erb]: @gandjhar what should I call this class then?

Comment (gandjhar) [inline app/views/welcome/index.html.erb]: Perhaps 'landing-cat'?  For the class name.

Comment (wearethefoos) [inline app/views/welcome/index.html.erb]: Alright, I'll do that! :+1:

**Comment by wearethefoos** [comment]
:shipit:

**Comment by wearethefoos** [comment]
:ship: :it:

**Comment by gandjhar** [comment]
Awesome work!  Make it so...

**Pull request merged**